server/public/model/config.go

```diff
@@ -933,27 +933,32 @@ func (s *ServiceSettings) SetDefaults(isUpdate bool) {
 }
 
 type CacheSettings struct {
-	CacheType     *string `access:",write_restrictable,cloud_restrictable"`
-	RedisAddress  *string `access:",write_restrictable,cloud_restrictable"` // telemetry: none
-	RedisPassword *string `access:",write_restrictable,cloud_restrictable"` // telemetry: none
-	RedisDB       *int    `access:",write_restrictable,cloud_restrictable"` // telemetry: none
+	CacheType          *string `access:",write_restrictable,cloud_restrictable"`
+	RedisAddress       *string `access:",write_restrictable,cloud_restrictable"` // telemetry: none
+	RedisPassword      *string `access:",write_restrictable,cloud_restrictable"` // telemetry: none
+	RedisDB            *int    `access:",write_restrictable,cloud_restrictable"` // telemetry: none
```

Comment:
Unrelated, but are we sanitizing any of these when returning the config to a client?

Reply:
We aren't returning any of these to the client. A client config setting needs to be manually added in `func GenerateClientConfig`.

Reply:
@agnivade Surely this is going to be in `/api/v4/config` API handler response though, along with everything else.

Reply:
Ah yes of course. Corrected that. Thanks.